src/main/resources/application.yml

```diff
@@ -23,14 +23,15 @@ sqs:
   process-print-request-batch-queue-name: ${SQS_PROCESS_PRINT_REQUEST_BATCH_QUEUE_NAME}
   process-print-response-file-queue-name: ${SQS_PROCESS_PRINT_RESPONSE_FILE_QUEUE_NAME}
   process-print-response-queue-name: ${SQS_PROCESS_PRINT_RESPONSE_QUEUE_NAME}
+  application-removed-queue-name: ${SQS_APPLICATION_REMOVED_QUEUE_NAME}
 
 api:
   print-api:
     generate-temporary-certificate:
       valid-on-date:
         max-calendar-days-in-future: 30
     retention.period:
-      certificate-delivery-info: "P28D" # 28 working days
+      certificate-initial-retention-period: "P28D" # 28 working days
```

Comment:
Could this be simplified `retention.period.certificate-initial-retention-period`?

Reply:
You mean place on one line? I was thinking other retention periods could be added in the future - e.g. for temporary certs.